src/foam/dao/EasyDAO.js

```diff
@@ -228,6 +228,23 @@ foam.CLASS({
         if ( getLastModifiedByAware() )
           delegate = new foam.nanos.auth.LastModifiedByAwareDAO.Builder(getX()).setDelegate(delegate).build();
 
+        if ( getApprovableAware() && getApprovableAwareEnabled() )
+          delegate = new foam.nanos.approval.ApprovableAwareDAO
+          .Builder(getX())
+          .setDaoKey(getName())
+          .setOf(getOf())
+          .setDelegate(delegate)
+          .build();
+
+        if ( getApprovableAware() && ! getApprovableAwareEnabled() )
+          delegate = new foam.nanos.approval.ApprovableAwareDAO
+          .Builder(getX())
+          .setDaoKey(getName())
+          .setOf(getOf())
+          .setDelegate(delegate)
+          .setIsEnabled(false)
```

Comment:
Can you combine these and just set isEnabled(getApprovableAwareEnabled())?

Reply:
Done.